### PR TITLE
Af 240 make variables internal

### PR DESCRIPTION
**Pull request by acollette**
This PR accomplishes the following:

=> Changes all variables from the following contracts to internal when technically feasible.
We can now decide if certain variables should be public if need be.

`DebtToken.sol`
`LendingPool.sol`
`Liquidator.sol`
`LendingPoolGuardian.sol`

=> Still to decide for following contracts if changes would be necessary or not (as limited impact)

`Tranche.sol`
`TrustedCreditor.sol`
`InterestRateModule.sol`

### CODE REVIEW

File: src/LendingPool.sol [hunk 1]
```diff
@@ -36,61 +36,61 @@ contract LendingPool is LendingPoolGuardian, TrustedCreditor, DebtToken, Interes
...
-    mapping(address => uint256) public realisedLiquidityOf;
+    mapping(address => uint256) internal realisedLiquidityOf;
     // Map Account => initiator.
     // Stores the address of the initiator of an auction, used to pay out the initiation fee after auction is ended.
-    mapping(address => address) public liquidationInitiator;
+    mapping(address => address) internal liquidationInitiator;
     // Map Account => owner => beneficiary => amount.
     // Stores the credit allowances for a beneficiary per Account and per Owner.
-    mapping(address => mapping(address => mapping(address => uint256))) public creditAllowance;
+    mapping(address => mapping(address => mapping(address => uint256))) internal creditAllowance;
```
Comment (Thomas-Smets) [inline src/LendingPool.sol:93]: This one should remain public

File: src/LendingPool.sol [hunk 2]
```diff
@@ -36,61 +36,61 @@ contract LendingPool is LendingPoolGuardian, TrustedCreditor, DebtToken, Interes
...
-    mapping(address => bool) public isTranche;
+    mapping(address => bool) internal isTranche;
     // Map tranche => interestWeight.
     // Fraction (interestWeightTranches[i] / totalInterestWeight) of the interest fees that go to Tranche i.
-    mapping(address => uint256) public interestWeight;
+    mapping(address => uint256) internal interestWeight;
     // Map tranche => realisedLiquidity.
     // Amount of `underlying asset` that is claimable by the Tranche. Does not take into account pending interests.
-    mapping(address => uint256) public realisedLiquidityOf;
+    mapping(address => uint256) internal realisedLiquidityOf;
```
Comment (Thomas-Smets) [inline src/LendingPool.sol:87]: This one should remain public

File: test/utils/Extensions.sol [hunk 3]
```diff
@@ -57,6 +69,9 @@ contract InterestRateModuleExtension is InterestRateModule {
 ////////////////////////////////////////////////////////////// */
 
 contract LendingPoolExtension is LendingPool {
+    // Seconds per year, leap years ignored.
+    uint256 public constant YEARLY_SECONDS_ = 31_536_000;
```
Comment (Thomas-Smets) [inline test/utils/Extensions.sol:73]: Not possible with a getter?

File: src/LendingPool.sol (hunk 1 above)
Comment (acollette) [reply src/LendingPool.sol]: done

File: test/fuzz/Fuzz.t.sol [hunk 4]
```diff
@@ -61,6 +62,7 @@ abstract contract Fuzz_Lending_Test is Base_Lending_Test, Fuzz_Test {
         asset = new Asset("Asset", "ASSET", 18);
         liquidator = new LiquidatorExtension(address(factory));
         pool = new LendingPoolExtension(asset, treasury, address(factory), address(liquidator));
+        poolTest = new LendingPool(asset, treasury, address(factory), address(liquidator));
```
[the remark this answers is not in the document]
Comment (acollette) [reply test/fuzz/Fuzz.t.sol]: not used anymore, deleted.

File: test/utils/Extensions.sol (hunk 3 above)
Comment (acollette) [reply test/utils/Extensions.sol]: done

File: src/LendingPool.sol (hunk 2 above)
Comment (acollette) [reply src/LendingPool.sol]: done

**Review by Thomas-Smets — APPROVED** [review]
gtg

**Pull request merged**